client/src/containers/CategoryForm.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+class CategoryForm extends Component {
```

Comment:
it's going to feel like i'm a `class` racist cause this also doesn't need to be a class, but i guess we could keep it for code readability for your renderList function.

Reply:
This is a container that's going to be responsible for part of the state once reduxForm is set up, that's why we set it up as a class component 😃